src/components/LayoutTree.tsx

```diff
@@ -358,13 +358,26 @@ const LayoutTree = () => {
   const treeThoughts = useSelector(linearizeTree, _.isEqual)
   const fontSize = useSelector(state => state.fontSize)
   const dragInProgress = useSelector(state => state.dragInProgress)
+  const ref = useRef<HTMLDivElement | null>(null)
   const indentDepth = useSelector(state =>
     state.cursor && state.cursor.length > 2
       ? // when the cursor is on a leaf, the indention level should not change
         state.cursor.length - (hasChildren(state, head(state.cursor)) ? 2 : 3)
       : 0,
   )
 
+  const bullet = ref.current?.querySelector('[aria-label=bullet]')
+  const [bulletWidth, setBulletWidth] = useState<number | undefined>(0)
+
+  // set the bullet width only during drag or when simulateDrop is true
+  useLayoutEffect(() => {
+    const newBulletWidth =
+      dragInProgress || testFlags.simulateDrop ? bullet?.getBoundingClientRect().width : bulletWidth
+
+    setBulletWidth(newBulletWidth)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Please do not disable the linter. It's almost always a developer mistake. 

In the rare instance that it needs to be disabled, you must provide a sound justification in a comment above.

Reply:
Hmm, I assumed since I was setting setting `bulletWidth`, adding it as dependency would causing a render loop. But now that I think about it it might not cause it.